packages/@ama-sdk/schematics/schematics/typescript/core/openapi-codegen-typescript/src/main/resources/typescriptFetch/api/api.mustache

```diff
@@ -11,23 +11,29 @@ import { Api, ApiClient, ApiTypes, computePiiParameterTokens, isJsonMimeType, Re
   {{#operation}}
     {{#allParams}}
       {{#isEnum}}
-export type {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum = {{#trimPipe}}{{#allowableValues}}{{#values}}'{{.}}' | {{/values}}{{/allowableValues}}{{/trimPipe}};
+/** Enum {{paramName}} used in the {{classname}}'s {{nickname}} function parameter */
+export type {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum = {{#trimPipe}}{{#allowableValues}}{{#values}}'{{.}}' | {{/values}}{{/allowableValues}}{{/trimPipe}};
+
+/** @deprecated please use {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum instead, will be remove in Otter v12 */
+export type {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum = {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum;
       {{/isEnum}}
     {{/allParams}}
   {{/operation}}
   {{#operation}}
-export interface {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData {
+/** Parameters object to {{classname}}'s {{nickname}} function */
+export interface {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData {
     {{#allParams}}
   /** {{#description}}{{{description}}}{{/description}}{{^description}}{{#isArray}}List of {{#plurialize}}{{#noArrayInType}}{{{dataType}}}{{/noArrayInType}}{{/plurialize}}{{/isArray}}{{/description}} */
   '{{baseName}}'{{^required}}?{{/required}}{{#required}}{{#defaultValue}}?{{/defaultValue}}{{/required}}: {{#isEnum}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}{{#uppercaseFirst}}{{paramName}}{{/uppercaseFirst}}Enum{{#isArray}}[]{{/isArray}}{{/isEnum}}{{^isEnum}}{{#isBodyParam}}{{{baseType}}}{{#isArray}}[]{{/isArray}}{{/isBodyParam}}{{^isBodyParam}}{{{dataType}}}{{/isBodyParam}}{{/isEnum}};
     {{/allParams}}
 }
   {{/operation}}
   {{#operation}}
-/**
- * @Deprecated, please use {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData
- */
-export interface {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}} extends {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData {}
+/** @deprecated please use {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData instead, will be remove in Otter v12 */
+export interface {{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData extends {{classname}}{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData {}
```

Comment:
even if the interface is deprecated it is still exported, and I am afraid that we will still have duplications on the barrel file, no ?  
`{{#uppercaseFirst}}{{nickname}}{{/uppercaseFirst}}RequestData` from one `my.api.ts` file and the same maybe from another file `your.api.ts`

Reply:
Yes, unfortunately the multi tag support cannot be achieved without breaking change.
So this PR is an enabler but the support will be effective only after breaking change.